pkg/metrics/exporter/export.go

```diff
@@ -69,6 +88,17 @@ func (e *Exporter) ExportFsMetrics(m *types.FsMetrics, imageRef string) error {
 	return e.output()
 }
 
+func ExportNydusdEventMetric(daemonID string, event string) error {
```

Comment:
Perhaps, we can rename this function to RecordDaemonEvent

Reply:
Export means the metrics is consumed and passed to another subsystem

Reply:
Yes, it is right. I have Fixed it.